R/tdata.R

```diff
@@ -0,0 +1,52 @@
+#' @export
```

Comment:
some tests for these functions will be nice.
Please add roxygen2 docs as are exported. 

tdata2env could give bad error outside the reactive context, should we use isolate or add some validation?

Reply:
yup all coming 👍